apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Limit Global Styles on WP.com to paid plans.
+ *
+ * @package full-site-editing-plugin
+ */
+
+/**
+ * Checks if Global Styles should be limited on the given site.
+ *
+ * @param  int $blog_id Blog ID.
+ * @return bool Whether Global Styles are limited.
+ */
+function wpcom_should_limit_global_styles( $blog_id = 0 ) {
+	if ( ! $blog_id ) {
+		$blog_id = get_current_blog_id();
+	}
+
+	// Do not limit Global Styles on Atomic sites for now, because blog stickers are not exposed
+	// to these sites and the project is still in a development stage that requires sites to have
+	// a certain blog sticker before restricting the feature. This is a temporary check that will
+	// be removed as part of the public launch.
+	if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
+		return false;
+	}
+
+	// Do not limit Global Styles on sites created before we made it a paid feature. This cutoff
+	// blog ID needs to be updated as part of the public launch.
+	if ( $blog_id < 210494207 ) {
+		return false;
+	}
+
+	if ( wpcom_site_has_feature( WPCOM_Features::GLOBAL_STYLES, $blog_id ) ) {
+		return false;
+	}
+
+	// During the development stage, we only limit Global Styles on sites that have opted in. This
+	// is a temporary check that will be removed as part of the public launch.
+	return has_blog_sticker( 'wpcom-limit-global-styles', $blog_id );
+}
+
+/**
+ * Enqueues the WP.com Global Styles scripts and styles.
+ *
+ * @return void
+ */
+function wpcom_global_styles_enqueue_scripts_and_styles() {
+	$screen = get_current_screen();
+	if ( ! $screen || 'site-editor' !== $screen->id ) {
+		return;
+	}
+
+	if ( ! wpcom_should_limit_global_styles() ) {
+		return;
+	}
+
+	$asset_file   = plugin_dir_path( __FILE__ ) . 'dist/wpcom-global-styles.asset.php';
+	$asset        = file_exists( $asset_file )
+		? require $asset_file
+		: null;
+	$dependencies = isset( $asset['dependencies'] ) ?
+		$asset['dependencies'] :
+		array();
```

Comment:
[not blocking]
We don't actually need the ternary for dependencies, we can replace the highlighted code with:
```
$dependencies = $asset['dependencies'] ?? array();
```